### PR TITLE
Allowing the player to mouse drop themselves out of an inflatable shelter

### PR DESCRIPTION
**Pull request by DeityLink**
Seriously why is that not a thing?

:cl:
* rscadd: You can now mousedrop yourself out of an inflatable shelter

### CODE REVIEW

File: code/game/objects/structures/inflatable.dm [hunk 1]
```diff
@@ -444,7 +444,7 @@
 			A.ex_act(severity++)
 	..()
 
-/obj/structure/inflatable/shelter/container_resist(mob/user)
+/obj/structure/inflatable/shelter/container_resist(var/mob/user,var/turf/dest)
```
Comment (Exxion) [inline code/game/objects/structures/inflatable.dm:447]: We generally prefer not using `var/` here

File: code/game/objects/structures/inflatable.dm [hunk 2]
```diff
@@ -495,6 +498,22 @@
 		if(do_after_many(user,list(target,src),20)) //Twice the normal time
 			enter_shelter(target)
 
+
+/obj/structure/inflatable/shelter/MouseDropFrom(over_object, src_location, var/turf/over_location, src_control, over_control, params)
```
Comment (Exxion) [inline code/game/objects/structures/inflatable.dm]: Ditto. Also why do you only have it on one of the args?

File: code/game/objects/structures/inflatable.dm (hunk 1 above)
Comment (ShiftyRail) [reply code/game/objects/structures/inflatable.dm:447]: (respectfully disagree)

Comment (DamianX) [reply code/game/objects/structures/inflatable.dm:447]: Is it finally time to write a style guide??

Comment (jknpj) [reply code/game/objects/structures/inflatable.dm:447]: do it

Comment (DeityLink) [reply code/game/objects/structures/inflatable.dm:447]: Been doing that for 5+ years, why the change?

File: code/game/objects/structures/inflatable.dm (hunk 2 above)
Comment (DeityLink) [reply code/game/objects/structures/inflatable.dm]: dirty copypasta from mecha.dm inbetween two rounds

File: code/game/objects/structures/inflatable.dm (hunk 1 above)
Comment (Exxion) [reply code/game/objects/structures/inflatable.dm:447]: It's just unnecessary, just like `src.`

File: code/game/objects/structures/inflatable.dm (hunk 2 above)
Comment (DeityLink) [reply code/game/objects/structures/inflatable.dm]: I removed it from this proc so it doesn't stick out like a sore thumb. Is that good enough?

File: code/game/objects/structures/inflatable.dm [hunk 3]
```diff
@@ -495,6 +498,22 @@
 		if(do_after_many(user,list(target,src),20)) //Twice the normal time
 			enter_shelter(target)
 
+
+/obj/structure/inflatable/shelter/MouseDropFrom(over_object, src_location, over_location, src_control, over_control, params)
```
Comment (Exxion) [inline code/game/objects/structures/inflatable.dm]:
```suggestion
/obj/structure/inflatable/shelter/MouseDropFrom(over_object, src_location, turf/over_location, src_control, over_control, params)
```

**Comment by Kurfursten** [comment]
Update /examine()

For perhaps obvious reasons, inflatable shelters have some of the most verbose explanation text of any object

**Comment by DeityLink** [comment]
done

**Pull request merged**